src/transforms/encryption-fallback-test.js

```diff
@@ -0,0 +1,91 @@
+const encryption = require('./encryption');
```

Comment:
Problem with this example is your are not testing the stream part, ie the crypted bytes can be cut anywhere